README_Instructions.md

```diff
@@ -0,0 +1,32 @@
+# EdgeX-TAF execution in Pipeline
+
+This is created as an intro to understand how EdgeX-TAF will be executed in pipeline
+
+## trigger.sh
+
+This is main script which will be triggered from pipeline.
+
+user will need to run script as below from pipeline
+
+sh trigger.sh --service functionalTest/V2-API/<service_name> --auth <username/password> --port <port_number> --module <module> --ip <IP_address>
+
+   * service_name: this will be service name which is present under V2-API
+   * username/password: this will be username and password of board. if user will not provide then it will
+                        take default as admin/admin
+   * port: this is socket port. as of now we will take as 8002
+   * module: this will be any module like KVM or STM
```

Comment:
Calling this input 'module' doesn't quite feel right.  Is something like 'dut' (for Device Under Test) or 'target' more appropriate?

@rajatkumar-eaton  - I am interested in your thoughts.

Reply:
@markabrezovec-eaton I will put name as "target"

Reply:
> Calling this input 'module' doesn't quite feel right. Is something like 'dut' (for Device Under Test) or 'target' more appropriate?
> 
> @rajatkumar-eaton - I am interested in your thoughts.

Yes @markabrezovec-eaton I completely agree on this. We should use DUT or Target Device or simply Target.

Reply:
Ok, I will make it as a Target.